### ISSUE TITLE
Running for a past period?

### ISSUE THREAD

**Issue by mrjcleaver** [ttscoff/Slogger#349]
Hi,

Is there a way to run slogger for, say, every eight hours over the period of last week?

Or for say, the month of November last year?

Thanks, Martin.

**Comment by ttscoff** [comment]
Not presently, no. You can run if for a certain period of time back to 
the present, but you can't put a to: date on it.

-Brett

**Comment by mrjcleaver** [comment]
Thanks Brett,

Not particularly hard though, is it?

Does the Day One API allow us to post-date? Would adding this as a feature mess up anything else in Slogger?

Thanks, M.

**Comment by ttscoff** [comment]
No, not really. There are some issues with dealing with date handling 
from various sources, and each plugin would have to implement the 
feature separately, but I can't think of any showstoppers (other than I 
have no time to put into it, so someone else would need to code it…).

-Brett

**Comment by mrjcleaver** [comment]
Do plugins already have sort of capabilities mechanism, whereby they can declare that they are capable of something? We'd need attributes such as capable of logging arbitrary dates.
